code_examples/sdk_examples/src/dapp/dapp/04_attest_credential.ts

```diff
@@ -12,14 +13,15 @@ export async function main({
   domainLinkageCredential: Kilt.ICredential
 }) {
   const api = Kilt.ConfigService.get('api')
-
   const { cTypeHash, claimHash } = Kilt.Attestation.fromCredentialAndDid(
     domainLinkageCredential,
     didUri
   )
   const attestationTx = api.tx.attestation.add(claimHash, cTypeHash, null)
 
   // We authorize the call using the attestation key of the Dapps DID.
+  const extrinsic = api.tx.did.dispatchAs(dappAccount.address, attestationTx)
+
   const submitTx = await Kilt.Did.authorizeTx(
```

Comment:
that bit is not in use any more now, am I right?

Reply:
Looks like yes.